packages/contracts/migrations/development/Deploy001_Diamond_Dollar.s.sol

```diff
@@ -0,0 +1,335 @@
+        ds.supportedInterfaces[type(IDiamondCut).interfaceId] = true;
+        ds.supportedInterfaces[type(IDiamondLoupe).interfaceId] = true;
+        ds.supportedInterfaces[type(IERC173).interfaceId] = true;
+
+        LibAccessControl.grantRole(DEFAULT_ADMIN_ROLE, _args.admin);
+        LibAccessControl.grantRole(DOLLAR_TOKEN_MINTER_ROLE, _args.admin);
+        LibAccessControl.grantRole(DOLLAR_TOKEN_BURNER_ROLE, _args.admin);
+        LibAccessControl.grantRole(PAUSER_ROLE, _args.admin);
+
+        AppStorage storage appStore = LibAppStorage.appStorage();
+        appStore.paused = false;
+        appStore.treasuryAddress = _args.admin;
+
+        // reentrancy guard
+        _initReentrancyGuard();
+    }
+}
+
+/**
+ * @notice Migration contract
+ * @dev Initial production migration includes the following contracts:
+ * - Dollar token
+ * - Governance token (already deployed https://etherscan.io/address/0x4e38D89362f7e5db0096CE44ebD021c3962aA9a0)
+ * - UbiquityPool (which is a facet of the Diamond contract)
+ *
+ * So we're deploying only contracts and facets necessary for features
+ * connected with the contracts above. Hence we omit the following facets
+ * from deployment:
+ * - BondingCurveFacet (used for UbiquiStick NFT, "on hold" now)
+ * - ChefFacet (staking is not a part of the initial deployment)
+ * - CollectableDustFacet (could be useful but not a must now)
+ * - CreditClockFacet (used for CreditNft which is not a part of the initial deployment)
+ * - CreditNftManagerFacet (used for CreditNft which is not a part of the initial deployment)
+ * - CreditNftRedemptionCalculatorFacet (used for CreditNft which is not a part of the initial deployment)
+ * - CreditRedemptionCalculatorFacet (used for Credit token which is not a part of the initial deployment)
+ * - CurveDollarIncentiveFacet (deprecated)
+ * - DollarMintCalculatorFacet (used for Dollar/Credit mint/burn mechanism which is not a part of the initial deployment)
+ * - DollarMintExcessFacet (used for Dollar/Credit mint/burn mechanism which is not a part of the initial deployment)
+ * - StakingFacet (staking is not a part of the initial deployment)
+ * - StakingFormulasFacet (staking is not a part of the initial deployment)
+ */
+contract Deploy001_Diamond_Dollar is Script, DiamondTestHelper {
+    // Dollar related contracts
+    UbiquityDollarToken public dollarToken;
+    ERC1967Proxy public proxyDollarToken;
+
+    // diamond related contracts
+    Diamond diamond;
+    DiamondInit diamondInit;
+
+    // diamond facet implementation instances (should not be used directly)
+    AccessControlFacet accessControlFacetImplementation;
+    DiamondCutFacet diamondCutFacetImplementation;
+    DiamondLoupeFacet diamondLoupeFacetImplementation;
+    ManagerFacet managerFacetImplementation;
+    OwnershipFacet ownershipFacetImplementation;
+    TWAPOracleDollar3poolFacet twapOracleDollar3PoolFacetImplementation;
+    UbiquityPoolFacet ubiquityPoolFacetImplementation;
```

Comment:
The UbiquityPool facet lacks some view methods i think it would be nice to have a view method for the `addCollateral()` and return the token that was added, this is not available as per testing.. i think if we agree, on another PR obviously, just worth noting

Reply:
> The UbiquityPool facet lacks some view methods

What methods exactly?

> it would be nice to have a view method for the addCollateral() and return the token that was added

You can get all collateral token addresses via the [allCollaterals()](https://github.com/ubiquity/ubiquity-dollar/blob/e8860cc5846442374d925278b9a68c05160ffd05/packages/contracts/src/dollar/facets/UbiquityPoolFacet.sol#L20C14-L20C28) view method along with [collateral info](https://github.com/ubiquity/ubiquity-dollar/blob/e8860cc5846442374d925278b9a68c05160ffd05/packages/contracts/src/dollar/facets/UbiquityPoolFacet.sol#L25)

Reply:
I confirm